index.js

```diff
@@ -40,7 +40,8 @@ function Argv (processArgs, cwd) {
     )
   }
 
-  var options
+  self.level = 1;
```

Comment:
I need the level to be modified incrementally, don't think it would be possible with a public here. It needs to follow on the object instance.